setup.py

```diff
@@ -23,25 +23,9 @@
 version = "4.3.0.dev0"
 
 import os
-import sys
 from setuptools import setup, find_packages
 
-if sys.version_info < (2, 6):
-    print("This version of ZODB requires Python 2.6 or higher")
-    sys.exit(0)
-
-if (3,) < sys.version_info < (3, 2):
-    print("This version of ZODB requires Python 3.2 or higher")
-    sys.exit(0)
-
-
-# The (non-obvious!) choices for the Trove Development Status line:
-# Development Status :: 5 - Production/Stable
-# Development Status :: 4 - Beta
-# Development Status :: 3 - Alpha
-
 classifiers = """\
-Development Status :: 4 - Beta
```

Comment:
OK.  FWIW, I read this as a signal about the state of the *project*, rather than a given release.